.github/workflows/cmake-multi-platform.yml

```diff
@@ -71,3 +72,33 @@ jobs:
     - name: Build
       # Build your program with the given configuration. Note that --config is needed because the default Windows generator is a multi-config generator (Visual Studio generator).
       run: cmake --build ${{ steps.strings.outputs.build-output-dir }} --config ${{ matrix.build_type }}
+
+    - uses: benjlevesque/short-sha@v2.2
+      id: short-sha
+      with:
+        length: 6
+
+    - if: matrix.os == 'windows-latest'
+      name: Zip binaries
+      run: >
+        git archive --format=zip -o golded-${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.c_compiler }}-${{ steps.short-sha.outputs.sha }}.zip
+        --add-file=${{ steps.strings.outputs.build-output-dir }}/golded3/Release/golded.exe
+        --add-file=${{ steps.strings.outputs.build-output-dir }}/goldnode/Release/goldnode.exe
+        --add-file=${{ steps.strings.outputs.build-output-dir }}/rddt/Release/rddt.exe
+        HEAD README.md
+
+    - if: matrix.os != 'windows-latest'
+      name: Zip binaries
+      run: >
+        git archive --format=zip -o golded-${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.c_compiler }}-${{ steps.short-sha.outputs.sha }}.zip
+        --add-file=${{ steps.strings.outputs.build-output-dir }}/golded3/golded
```

Comment:
isn't it gedlnx?

Reply:
I have gedlnx, gnlnx and rddtlnx

Reply:
It's not, when build using cmake.

Reply:
Why don't they produce the same artifacts? :(

Reply:
legacy make did this, cmake is making straightforward names

Reply:
all done https://github.com/golded-plus/golded-plus/actions/runs/6630863513
in the summary there is archive of all the artifacts from the builds

Reply:
I'm pretty sure there were reasons to give these names :)